File: CSharp/Supermarket/Model/Checkout/BookingLine.cs
namespace Supermarket.Model.Checkout
{
    public class BookingLine
    {
        public BookingLine(string text, decimal price)
        {
            Text = text;
            Price = price;
        }

        public string Text { get; }

        public decimal Price { get; }

        public override string ToString() => $"{Text}: {Price:C}";
    }
}

File: java/src/main/java/supermarket/model/checkout/BookingLine.java
package supermarket.model.checkout;

import java.math.BigDecimal;

public class BookingLine {
	private String text;
	private BigDecimal price;

	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}

	public BigDecimal getPrice() {
		return price;
	}

	public void setPrice(BigDecimal price) {
		this.price = price;
	}

	@Override
	public String toString() {
		return String.format("%s: %d", text, price);
	}
}
